src/main/java/com/jalasoft/webservice/db/QueryManager.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
```

Comment:
This space line does'nt need to be here.

Reply:
Done